packages/radix-vue/src/TagsInput/TagsInputRoot.vue

```diff
@@ -142,7 +142,7 @@ provideTagsInputRootContext({
   },
   onRemoveValue: (index) => {
     if (index !== -1)
-      modelValue.value.splice(index, 1)
+      modelValue.value = modelValue.value.toSpliced(index, 1)
```

Comment:
Let's not use [`toSpliced`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSpliced#browser_compatibility) yet.

Maybe let's do 
```suggestion
      modelValue.value = [...modelValue.value].splice(index, 1)
```